Makefile

```diff
@@ -43,7 +53,7 @@ docker-context: scm-source.json linux
 	cp build/linux/${BINARY} scm-source.json docker/build/
 
 docker: ${DOCKERFILE} docker-context
-	cd docker && docker build --rm -t "$(IMAGE):$(TAG)" .
+	docker build --rm -t "$(IMAGE):$(TAG)" -f "${DOCKERFILE}" .
```

Comment:
Maybe add a `-debug` suffix to the tag just so it's more visible?

Reply:
Also: note this changes behavior a bit.  As you no longer `cd` into the `docker` directory, the Docker build context will be the top level directory of the repository, so more things will be copied to the Docker daemon for the build.  Also, the `COPY build/* /` instruction from the `Dockerfile` will actually be using `$repo_root/build` instead of the `$repo_root/docker/build` directory prepared by the recipe in the `docker-context` target: `mkdir -p docker/build; cp build/linux/${BINARY} scm-source.json docker/build/`.  I'm not sure if the binary would even be present in the root directory of the image at `/postgres-operator`.  Did you intend this?

Reply:
Yeah, it makes sense.